src/services/goToDefinition.ts

```diff
@@ -510,4 +522,25 @@ namespace ts.GoToDefinition {
                 return false;
         }
     }
+
+    interface StatementWithExpression extends Statement {
+        readonly expression: Expression
+    }
+
+    function createDefinitionInfoFromStatement<K extends KeywordSyntaxKind>(statement: StatementWithExpression, keywordKind: K, sourceFile: SourceFile, name: string): DefinitionInfo {
+        const keyword = find(statement.getChildren(sourceFile), (node): node is Token<K> => node.kind === keywordKind)!;

```

Comment:
```suggestion
        const keyword = find(statement.getChildren(sourceFile), node => node.kind === keywordKind)!;
```

I don't think you need the type predicate signature, do you?

Reply:
fixed